v63003/u_inref/gtm8767.csh

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+set randint = $$
+set randhex = `openssl rand -hex 3`
+set randnum = $$
+
+$gtm_tst/com/dbcreate.csh mumps 1 >>& create1.out
+echo "Setting Hard Spin Count"
```

Comment:
AReplace this line with

echo "# Setting Hard Spin Count to a random value using MUPIP SET -HARD_SPIN_COUNT"

Also do similar change in other places where MUPIP SET is invoked.